src/main/java/com/rettichlp/UnicacityAddon/commands/faction/ReinforcementCommand.java

```diff
@@ -77,7 +82,15 @@ public void execute(@Nonnull MinecraftServer server, @Nonnull ICommandSender sen
             p.sendChatMessage(chatType.getChatCommand() + " " + name + ", ich bin zu deinem Verstärkungsruf unterwegs! (" + (int) p.getPosition().getDistance(x, y, z) + " Meter entfernt)");
             NavigationUtils.stopRoute();
             p.setNaviRoute(x, y, z);
-
+            if (ConfigElements.automaticReinfscreen()) {
+                try {
+                    File file = FileManager.getNewImageFile();
```

Comment:
Ich würde den Screenshot vielleicht sogar im Reinforcement Ordner speichern
```suggestion
                    File file = FileManager.getNewActivityImageFile("reinforcement");
```